framework/tests/all-systems/java8inference/Bug11.java

```diff
@@ -0,0 +1,21 @@
+package inference.guava;
+
+import java.io.Serializable;
+import java.util.EnumMap;
+import java.util.Map;
+
+public class Bug11 {
+
+    public static <K1, V1> MyMap<K1, V1> copyOf(Map<? extends K1, ? extends V1> map) {
+        @SuppressWarnings("unchecked")
+        MyMap<K1, V1> kvMap = (MyMap<K1, V1>) copyOfEnumMap((EnumMap<?, ?>) map);
```

Comment:
@smillst  This is failing with:

```
    1 unexpected diagnostic was found:
    Bug11.java:11: error: (type.argument.type.incompatible)
```

Reply:
This test case is really just "crash checking", so I suppressed all warnings.  Thanks for pointing this out!